nomad/grapl-generator.nomad

```diff
@@ -0,0 +1,72 @@
+variable "tenant_id" {
+    type = string
+    description = "The opaque tenant id."
```

Comment:
What does "opaque" mean here? This is just the tenant UUID right?

Reply:
Yup tenant UUID

Reply:
let's update the name to tenant_uuid or update the description?